api/swagger.yaml

```diff
@@ -4040,6 +4040,13 @@ definitions:
         enum: ["cgroupfs", "systemd", "none"]
         default: "cgroupfs"
         example: "cgroupfs"
+      CgroupVersion:
+        description: |
+          The version of the cgroup.
+        type: "integer"
+        enum: [1, 2]
```

Comment:
Discussing with @cpuguy83 @SamWhited if we should just use a string for this; it may be more flexible

Reply:
Seems safer; if we don't they'll end up releasing `1.2-beta` and accidentally releasing it.

Reply:
updated